ledger/complete/mtrie/trie/trie.go

```diff
@@ -464,25 +482,25 @@ func update(
 		}
 	}
 
-	// set the parent node children
-	var lchildParent, rchildParent *node.Node
```

Comment:
The "childParent" is confusing. Since we are updating the trie by creating new nodes, I think using "old" and "new" would be better.